.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,4 +11,4 @@ Resolves #NUMBER
 
 - [ ] I have assigned myself to this PR and the corresponding issues
 - [ ] Tests added for new features
```

Comment:
```suggestion
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
```

Reply:
Because saying tests and then referring only to manual testing is a bit confusing.